src/components/App.js

```diff
@@ -1,7 +1,7 @@
 import '../assets/css/App.css'
 import React, { Component } from 'react'
 
-class App extends React.Component {
+class App extends Component {
```

Comment:
Not related to dev tools changes but seems wrong to reference `Component` if already imported.